File: backend/src/main/java/com/campus/Campus_Connect/features/event/dto/response/ParticipantTeamResponse.java
package com.campus.Campus_Connect.features.event.dto.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.util.List;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class ParticipantTeamResponse {

    private Integer teamId;

    private String teamName;

    private String leaderName;

    private Integer memberCount;

    private List<TeamMemberResponse> members;

}

File: android-app/app/src/main/java/com/example/campusconnect/feature/events/data/remote/response/ParticipantTeamResponse.kt
package com.example.campusconnect.feature.events.data.remote.response

import com.google.gson.annotations.SerializedName

data class ParticipantTeamResponse(

    @SerializedName("teamId")
    val teamId: Int,

    @SerializedName("teamName")
    val teamName: String,

    @SerializedName("leaderName")
    val leaderName: String?,

    @SerializedName("memberCount")
    val memberCount: Int,

    @SerializedName("members")
    val members: List<TeamMemberResponse>
)